src/lib/__generated/graphql.schema.graphql

```diff
@@ -120,7 +120,7 @@ enum AssetOrder {
 }
 
 """
-These fields are used to add SEO related properties to the various pages that we render [See type definition](https://app.contentful.com/spaces/a67phq2m6waq/content_types/componentSeo)
+To have SEO-related properties in the pages we render [See type definition](https://app.contentful.com/spaces/k6wz64nn0wjv/content_types/componentSeo)
```

Comment:
i guess the spaceId here should not be changed?

Reply:
No it should not, fascinating that that changed.. let me look at that